__tests__/rtFixtureLoaderHelper.js

```diff
@@ -0,0 +1,107 @@
+const enumerationRt = require('./__fixtures__/ItemEnumeration.json')
+const chronologyRt = require('./__fixtures__/ItemChronology.json')
+const adminMetaRt = require('./__fixtures__/adminMetadata.json')
+const adminMetaStatusRt = require('./__fixtures__/adminMetadataStatus.json')
+const rdaItemMonoRt = require('./__fixtures__/rdaItemMonograph.json')
+const rdaManifestMonoRt = require('./__fixtures__/RDAMainifestationMonograph.json')
+
+export const resourceTemplateId2Json = [
+  { id: 'resourceTemplate:bf2:Monograph:Instance', json: monographInstanceRt },
+  { id: 'resourceTemplate:bf2:Monograph:Work', json: monographWorkRt },
+  { id: 'resourceTemplate:bf2:Identifiers:Barcode', json: barcodeRt },
+  { id: 'resourceTemplate:bf2:Note', json: noteRt },
+  { id: 'resourceTemplate:bf2:ParallelTitle', json: parallelTitleRt },
+  { id: 'resourceTemplate:bf2:Title', json: titleRt },
+  { id: 'resourceTemplate:bf2:Title:Note', json: titleNoteRt },
+  { id: 'resourceTemplate:bflc:TranscribedTitle', json: transcribedTitleRt },
+  { id: 'resourceTemplate:bf2:Title:VarTitle', json: varTitleRt },
+  { id: 'resourceTemplate:bf2:WorkTitle', json: workTitleRt },
+  { id: 'resourceTemplate:bf2:WorkVariantTitle', json: workVariantTitleRt },
+  { id: 'resourceTemplate:bf2:Identifiers:LCCN', json: lccnRt },
+  { id: 'resourceTemplate:bf2:Identifiers:DDC', json: ddcRt },
+  { id: 'resourceTemplate:bf2:Identifiers:Shelfmark', json: shelfMarkRt },
+  { id: 'resourceTemplate:bf2:Item', json: itemRt },
+  { id: 'resourceTemplate:bf2:Item:Retention', json: retentionRt },
+  { id: 'resourceTemplate:bf2:Item:ItemAcqSource', json: itemAcqSourceRt },
+  { id: 'resourceTemplate:bf2:Item:Enumeration', json: enumerationRt },
+  { id: 'resourceTemplate:bf2:Item:Chronology', json: chronologyRt },
+  { id: 'rt:bf2:AdminMetadata', json: adminMetaRt },
+  { id: 'rt:bf2:AdminMetadata:Status', json: adminMetaStatusRt },
+  { id: 'rt:rda:item:monograph', json: rdaItemMonoRt },
+  { id: 'rt:rda:manifestation:monograph', json: rdaManifestMonoRt },
+]
+
+const emptyTemplate = { propertyTemplates: [{}] }
+
+export const resourceTemplateIds = resourceTemplateId2Json.map(template => template.id)
+
+export const getFixtureResourceTemplate = (templateId) => {
+  if (!templateId) {
+    emptyTemplate.error = 'ERROR: asked for resourceTemplate with null/undefined id'
+
+    return emptyTemplate
+  }
+
+  if (!resourceTemplateIds.includes(templateId)) {
+    emptyTemplate.error = `ERROR: non-fixture resourceTemplate: ${templateId}`
+
+    return emptyTemplate
+  }
+
+  const fixtureResponse = { response: {} }
+
+  fixtureResponse.response.body = resourceTemplateId2Json.find((template) => {
+    if (template.id === templateId) return template
+  }).json
+
+  return new Promise((resolve) => {
+    resolve(fixtureResponse)
+  })
+}
+
+export const fixtureResourcesInGroupContainer = (group) => {
+  const container = `http://spoof.trellis.io/${group}`
```

Comment:
Should this be renamed to something like `http://fixures.trellis.io/${group}`?

Reply:
I thought about this.  However, in this case, the URL really is a spoof - it's a stand in for something real, and the server doesn't actually exist.  These fixtures have no true server home;  it could be "bogus.ldp.server."  I am happy to change it if folks prefer.

Reply:
It's just something to return in a fake response.body so it doesn't really matter. I'm fine with leaving it as-is.